src/message/html/HtmlTagImg.js

```diff
@@ -8,16 +8,11 @@ const styles = StyleSheet.create({
     width: 296,
     height: 150,
   },
-  emoji: {
-    width: 24,
-    height: 24,
-  },
 });
 
-export default ({ src, className, auth }) => (
+export default ({ src, style, auth }) => (
   <Image
     source={getResource(src, auth)}
-    resizeMode={className === 'emoji' ? 'cover' : 'contain'}
-    style={className === 'emoji' ? styles.emoji : styles.img}
+    style={[styles.img, style]}
```

Comment:
The problem occurs when an image is longer than it is wide, or wider than it is long. In that case, we need to use the 'contain' mode to ensure that the whole image is visible in the frame.